sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1316,9 +1317,9 @@ SUBROUTINE MDL2P(iostatusD3D)
             DO J=JSTA,JEND
               DO I=ISTA,IEND
                 GRID1(I,J) = SPVAL
-                CFRSL(I,J) = MIN(MAX(0.0,CFRSL(I,J)),1.0)
                 IF(abs(CFRSL(I,J)-SPVAL) > SMALL)                   &    
```

Comment:
@WenMeng-NOAA I think it should be:
                IF(abs(CFRSL(I,J)-SPVAL) > SMALL) then
                  CFRSL(I,J) = MIN(MAX(0.0,CFRSL(I,J)),1.0)
                  GRID1(I,J) = CFRSL(I,J)*H100
                endif
Otherwise, we wll have GRID1=spval*H00

Reply:
@junwang-noaa You are right. Thanks for catching it.